README.md

```diff
@@ -69,10 +70,27 @@ PrivateTmp=no
 [Install]
 WantedBy=multi-user.target
 ```
+
+#### Service overrites(dropin):
+```puppet
+systemd::dropin { 'ceph-disk@':
+  env_vars => ['CEPH_DISK_TIMEOUT=3000'],
+}
+```
+
+This is going to create the following overrrite file:
+
+```bash
+# cat /etc/systemd/system/ceph-disk@.service.d/override.conf:
+[Service]
+Environment=CEPH_DISK_TIMEOUT=3000
```

Comment:
an example showing a override using a reset of a list would be useful, I assume in this case it would be:

    env_vars => ['', 'CEPH_DISK_TIMEOUT=3000']

to get

    Environment=
    Environment=CEPH_DISK_TIMEOUT=3000